requirements/extras/sqlalchemy.txt

```diff
@@ -1 +1 @@
-sqlalchemy~=1.4.34
```

Comment:
Please avoid pinning to specific versions.
@auvipy Can you please fix this?

Reply:
pyup update them regularly. for sanity purpose this seems to be norm now